lagom-eventstore-example/README.md

```diff
@@ -0,0 +1,137 @@
+
+This project demonstrates a simple Lagom service that includes a [Read-Side](https://www.lagomframework.com/documentation/current/java/ReadSide.html) processor which stores events of the [Persistent Entities](https://www.lagomframework.com/documentation/1.3.x/java/PersistentEntity.html) into Project EventStore for further analysis.
+
+The Lagom service exposes a single HTTP endpoint to simulate events.
+
+## Prerequisites
+
+To build and run this example, you need:
+
+- [git](https://git-scm.com/)
+- [Java SE 8 JDK](http://www.oracle.com/technetwork/java/javase/overview/index.html)
+- [Maven 3.2.1+](https://maven.apache.org/) to build and run the Lagom project (3.5.0 recommended)
+- IBM Project EventStore [downloaded](https://github.com/IBMProjectEventStore/EventStore-DeveloperPreview/releases/tag/1.1.1), [installed](https://www.ibm.com/support/knowledgecenter/SSGNPV/eventstore/desktop/welcome.html) and running on your local machine. (This process requires 12+Gb of downloads)
+
+Once you have an IBM Project EventStore running in your machine, the main steps to run this example are:
+
+1.  [Download and set up the Lagom service](#download-and-set-up-the-lagom-service)
+2.  [Start the Lagom sample application](#start-the-lagom-sample-application)
+3.  [Generate some traffic on the Lagom service](#generate-some-traffic-on-the-lagom-service)
+4.  [Analyse the traffic using IBM Project EventStore Notebooks](#analyse-the-traffic-using-ibm-project-eventstore-notebooks)
+
+## Download and set up the Lagom service
+
+(Make sure you completed all the steps on the Pre-Requisites section and IBM Project EventStore is installed and running)
+
+Follow these steps to get a local copy of this project and configure it to connect it to the IBM Project EventStore running on your machine
+
+1.  Open a command line shell and clone this repository:
+    ```
+    git clone https://github.com/lagom/ibm-integration-examples.git
+    ```
+2.  Change into the root directory for this example:
+    ```
+    cd lagom-eventstore-example
+    ```
+3.  To supply the configuration, do one of the following:
+    1. Open the `lagom-eventstore-impl/src/main/resources/ibm-event-store.conf` file in a text editor and fill in the empty value of the `endpoints` setting.
+    2. On the same file, provide the value of the `db.name` setting. The `db.name` is not hardcoded because IBM Project EventStore can only hold one database at the same time. If you already have a database and its name is different from the name you want Lagom to use you will have to remove the database from the IBM Project EventStore manually. This example application may rebuild the EventStore from scratch on every reboot so you can test from a clean slate: use the `clear-schema` config to rebuild the database from scratch on every run.
```

Comment:
Here, too, I'm being asked to make a decision but I have no context. Maybe it could be reworded to clarify that leaving the default is usually OK, unless you've previously removed and recreated the database in your EventStore installation (is that right?)